src/entity/BookmarkList.ts

```diff
@@ -9,6 +20,26 @@ export class BookmarkList {
   @Index()
   userId: string;
 
+  @ManyToOne('User', {
+    lazy: true,
+    onDelete: 'CASCADE',
+  })
+  user: Promise<User>;
+
   @Column({ type: 'text' })
   name: string;
+
+  @Column({ type: 'text', nullable: true })
+  icon: string;
```

Comment:
should be icon?: string | null if its nullable

Reply:
Nice catch, updated!